### PR TITLE
Remove disclaimer and incubating

### PR DESCRIPTION
**Pull request by chengshiwen**

### CODE REVIEW

File: community/en-us/release.md [hunk 1]
```diff
@@ -226,10 +226,10 @@ cd ~/ds_svn/dev/dolphinscheduler/${RELEASE.VERSION}
 Add source code packages, binary packages and executable binary packages to SVN working directory.
 
 ```shell
-cp -f ~/incubator-dolphinscheduler/dolphinscheduler-dist/target/*.zip ~/ds_svn/dev/dolphinscheduler/${RELEASE.VERSION}
-cp -f ~/incubator-dolphinscheduler/dolphinscheduler-dist/target/*.zip.asc ~/ds_svn/dev/dolphinscheduler/${RELEASE.VERSION}
-cp -f ~/incubator-dolphinscheduler/dolphinscheduler-dist/target/*.tar.gz ~/ds_svn/dev/dolphinscheduler/${RELEASE.VERSION}
-cp -f ~/incubator-dolphinscheduler/dolphinscheduler-dist/target/*.tar.gz.asc ~/ds_svn/dev/dolphinscheduler/${RELEASE.VERSION}
+cp -f ~/dolphinscheduler/dolphinscheduler-dist/target/*.zip ~/ds_svn/dev/dolphinscheduler/${RELEASE.VERSION}
+cp -f ~/dolphinscheduler/dolphinscheduler-dist/target/*.zip.asc ~/ds_svn/dev/dolphinscheduler/${RELEASE.VERSION}
```
Comment (CalvinKirs) [inline community/en-us/release.md:230]: It seems that our current compression is in tar format.

File: community/en-us/release.md [hunk 2]
```diff
@@ -295,7 +295,6 @@ Decompress `apache-dolphinscheduler-incubating-${RELEASE.VERSION}-src.zip` and c
 
 *   Check whether source tarball is oversized for including nonessential files
 *   The release files have the word `incubating` in their name
-*   `DISCLAIMER` file exists
```
Comment (CalvinKirs) [inline community/en-us/release.md:298]: Both of these need to be deleted

File: community/en-us/release.md [hunk 3]
```diff
@@ -310,7 +309,6 @@ Decompress `apache-dolphinscheduler-incubating-${RELEASE.VERSION}-dolphinschedul
 to check the following items:
 
 - The release files have the word `incubating` in their name
-- `DISCLAIMER` file exists
```
Comment (CalvinKirs) [inline community/en-us/release.md:313]: Both of these need to be deleted

File: community/en-us/release.md [hunk 4]
```diff
@@ -354,7 +352,7 @@ Hello DolphinScheduler Community,
 This is a call for vote to release Apache DolphinScheduler (Incubating) version ${RELEASE.VERSION}
 
 Release notes:
-https://github.com/apache/incubator-dolphinscheduler/blob/${RELEASE.VERSION}/ReleaseNotes.md
+https://github.com/apache/dolphinscheduler/blob/${RELEASE.VERSION}/ReleaseNotes.md
 
 The release candidates:
 https://dist.apache.org/repos/dist/dev/incubator/dolphinscheduler/${RELEASE.VERSION}/
```
Comment (CalvinKirs) [inline community/en-us/release.md:358]: update:https://dist.apache.org/repos/dist/dev/dolphinscheduler/${RELEASE.VERSION}/

File: community/en-us/release.md [hunk 5]
```diff
@@ -454,7 +452,7 @@ https://lists.apache.org/thread.html/xxxxxxxxxxxxxxxxxxxxxxx
 https://lists.apache.org/thread.html/xxxxxxxxxxxxxxxxxxxxxxx
 
 Release notes:
-https://github.com/apache/incubator-dolphinscheduler/blob/${RELEASE.VERSION}/ReleaseNotes.md
+https://github.com/apache/dolphinscheduler/blob/${RELEASE.VERSION}/ReleaseNotes.md
 
 The release candidates:
 https://dist.apache.org/repos/dist/dev/incubator/dolphinscheduler/${RELEASE.VERSION}/
```
Comment (CalvinKirs) [inline community/en-us/release.md:458]: delete incubator

**Review by CalvinKirs — CHANGES_REQUESTED** [review]
There are still some missing

File: community/en-us/release.md (hunk 2 above)
Comment (chengshiwen) [reply community/en-us/release.md:298]: Done

File: community/en-us/release.md (hunk 3 above)
Comment (chengshiwen) [reply community/en-us/release.md:313]: Done

File: community/en-us/release.md (hunk 4 above)
Comment (chengshiwen) [reply community/en-us/release.md:358]: Done

File: community/en-us/release.md (hunk 5 above)
Comment (chengshiwen) [reply community/en-us/release.md:458]: Done

File: community/en-us/release.md (hunk 1 above)
Comment (chengshiwen) [reply community/en-us/release.md:230]: Done

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**